### PR TITLE
[Pipelines] Enable livelogs

### PR DESCRIPTION
**Pull request by shahiinn**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart Version bumped
- [ ] CHANGELOG.md updated
- [ ] Variables and other changes are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/pipelines/values.yaml [hunk 1]
```diff
@@ -193,6 +193,10 @@ pipelines:
     enabled: false
   ## Logging settings
   logging:
+    ## livelog settings
+    view:
+      ## if enabled, livelogs will be enabled
+      enabled: false
```
Comment (Bharath92) [inline stable/pipelines/values.yaml:199]: is this disabled by default?

File: stable/pipelines/values.yaml [hunk 2]
```diff
@@ -836,6 +840,13 @@ pipelines:
       metrics:
         enabled: {{ .Values.pipelines.metrics.enabled }}
       logging:
+
+        view:
+          enabled: {{ .Values.pipelines.logging.view.enabled }}
+          refreshRate: 10s
+          concurrentSessionsPerUser: 10
+          files: "router-service.log,router-request.log,marshaller.log,extensionSync.log,hookHandler.log,pipelineSync.log,runTrigger.log,templateSync.log,logup.log,api.log,stepTrigger.log,cron.log,www.log"
```
Comment (anastasiagrinman) [inline stable/pipelines/values.yaml:848]: @shahiinn need to assure what are the default files for pipelines

File: stable/pipelines/values.yaml (hunk 1 above)
Comment (anastasiagrinman) [reply stable/pipelines/values.yaml:199]: @shahiinn can you check it with @chukka and @JfrogPrasanna please?

File: stable/pipelines/values.yaml (hunk 2 above)
Comment (Bharath92) [inline stable/pipelines/values.yaml:848]: should this be added here? By default, in an on-prem install, users should be able to look at any log files right?

Comment (JfrogPrasanna) [reply stable/pipelines/values.yaml]: @rahulsadanandan No need to add files entry here. Pls remove.

File: stable/pipelines/values.yaml (hunk 1 above)
Comment (JfrogPrasanna) [reply stable/pipelines/values.yaml]: @rahulsadanandan default false is fine for 7.16

**Review by JfrogPrasanna — CHANGES_REQUESTED** [review]
Pls make changes as per comments.

File: stable/pipelines/values.yaml (hunk 1 above)
Comment (rahulsadanandan) [reply stable/pipelines/values.yaml:199]: It is set false now.

**Review by Bharath92 — APPROVED** [review]

File: stable/pipelines/values.yaml (hunk 2 above)
Comment (rahulsadanandan) [reply stable/pipelines/values.yaml]: Its removed

**Review by chukka — APPROVED** [review]

**Review by JfrogPrasanna — APPROVED** [review]
LGTM

**Review by eldada — APPROVED** [review]

**Pull request merged**